py3-elfdeps.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: py3-elfdeps
+  version: 0.2.0
+  epoch: 0
+  description: Python implementation of RPM elfdeps
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provider-priority: "0"
+
+vars:
+  module_name: elfdeps
+  pypi-package: elfdeps
+
+data:
+  - name: py-versions
+    items:
+      "3.10": "310"
+      "3.11": "311"
+      "3.12": "312"
+      "3.13": "300"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-supported-pip
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/python-wheel-build/elfdeps.git
```

Comment:
we have no linting for this; but most github.com urls are without ".git" suffix, note this url is used to generate PURLs and download urls, and sometimes visible as human clickable too. Thus prefer the without ".git". It then is also easier to parse.

Note this is pedantic/cosmetic request, as there are redirect via any protocol both ways between suffix-less and .git suffix urls.

Reply:
Done.